translations/es.json

```diff
@@ -1,181 +1,181 @@
-	"Configuring local host environment ...": "",
-	"Configuring {{.name}} (Container Networking Interface) ...": "",
-	"Confirm that you have a working internet connection and that your VM has not run out of resources by using: 'minikube logs'": "",
-	"Confirm that you have supplied the correct value to --hyperv-virtual-switch using the 'Get-VMSwitch' command": "",
-	"Connect to LoadBalancer services": "",
-	"Consider creating a cluster with larger memory size using `minikube start --memory SIZE_MB` ": "",
-	"Consider increasing Docker Desktop's memory size.": "",
-	"Could not determine a Google Cloud project, which might be ok.": "",
-	"Could not find any GCP credentials. Either run `gcloud auth application-default login` or set the GOOGLE_APPLICATION_CREDENTIALS environment variable to the path of your credentials file.": "",
-	"Could not process error from failed deletion": "",
-	"Could not process errors from failed deletion": "",
+	"Amount of time to wait for a service in seconds": "Cantidad de tiempo para esperar por un servicio en segundos",
+	"Amount of time to wait for service in seconds": "Cantidad de tiempo para esperar un servicio en segundos",
+	"Another hypervisor, such as VirtualBox, is conflicting with KVM. Please stop the other hypervisor, or use --driver to switch to it.": "Otro hipervisor, por ejemplo VirtualBox, está en conflicto con KVM. Por favor detén el otro hipervisor, o usa --driver para cambiarlo.",
+	"Another program is using a file required by minikube. If you are using Hyper-V, try stopping the minikube VM from within the Hyper-V manager": "Otro programa está usando un archivo requerido por minikube. Si estas usando Hyper-V, intenta detener la máquina virtual de minikube desde el administrador de Hyper-V",
+	"Automatically selected the {{.driver}} driver": "Controlador {{.driver}} seleccionado automáticamente",
+	"Automatically selected the {{.driver}} driver. Other choices: {{.alternates}}": "Controlador {{.driver}} seleccionado automáticamente. Otras opciones: {{.alternates}}",
+	"Available Commands": "Comandos disponibles",
+	"Basic Commands:": "Comandos basicos:",
+	"Because you are using a Docker driver on {{.operating_system}}, the terminal needs to be open to run it.": "Porque estás usando controlador Docker en {{.operating_system}}, la terminal debe abrirse para ejecutarlo.",
+	"Bind Address: {{.Address}}": "Dirección de enlace: {{.Address}}",
+	"Both driver={{.driver}} and vm-driver={{.vmd}} have been set.\n\n    Since vm-driver is deprecated, minikube will default to driver={{.driver}}.\n\n    If vm-driver is set in the global config, please run \"minikube config unset vm-driver\" to resolve this warning.\n\t\t\t": "Ambos driver={{.driver}} y vm-driver={{.vmd}} han sido establecidos.\n\n	vm-driver y es obsoleto, el por defecto de minikube será driver={{.driver}}.\n\n Si vm-driver está establecido en la configuracion global, ejecuta \"minikube config unset vm-driver\" para resolver esta advertencia.\n\t\t\t",
```

Comment:
This line has a tab character at the `\n\n	vm-driver y es obsoleto,`. Changing that tab character to spaces after the two newlines will make this valid json. Once that's fixed we can accept this PR.